scripts/run_cpp_examples.sh

```diff
@@ -36,6 +36,20 @@ do
     cd -
 done
 
+# MPI only examples
+if [[ $PREFIX = mpirun* ]]
```

Comment:
wouldn't it be nicer to add `remote` to the list of examples above in this case?

Reply:
That I do not understand?

Reply:
Lines 29 and following: same loop body, so lines 41ff are code repetition. Just a detail.

Reply:
Yeah. I decided not to go down this rabbit hole
https://stackoverflow.com/questions/8063228/check-if-a-variable-exists-in-a-list-in-bash
and instead copy paste the loop. Sue me ;)